sky/services/activity/activity.mojom

```diff
@@ -67,6 +69,10 @@ interface Activity {
   GetCacheDir() => (string path);
 };
 
+interface UpdateService {
+  NotifyUpdateCheckComplete();
```

Comment:
Can do. How do you feel about using a mojom interface for this over a native method?